floss/utils.py

```diff
@@ -541,6 +542,11 @@ def get_static_strings(sample: Path, min_length: int) -> list:
     """
     Returns list of static strings from the file which are above the minimum length
     """
+
+    if os.stat(sample).st_size == 0:
+        logger.error("File is empty")
```

Comment:
i think this should be a warning at most, not an error (something that cannot be recovered).